### PR TITLE
Make TaggableManager.set API compatible with Django's RelatedManager.set

### PR DESCRIPTION
**Pull request by pablolmedorado**
See #686

### CODE REVIEW

**Review by auvipy — CHANGES_REQUESTED** [review]
please fix the conflicts and rebase issues

File: docs/api.rst [hunk 1]
```diff
@@ -36,7 +36,7 @@ playing around with the API.
 
         Removes all tags from an object.
 
-    .. method:: set(*tags, through_defaults=None, clear=False)
+    .. method:: set(tags, through_defaults=None, clear=False)
```
Comment (auvipy) [inline docs/api.rst:39]: def set(self, tags, *, through_defaults=None, **kwargs) has separate *

Comment (pablolmedorado) [reply docs/api.rst]: Done! Nice spot

**Comment by rtpg** [comment]
I don't consider this a blocker but I think it would be good if the release notes just went into more detail, saying that this changed from taking varargs to taking an explicit list (perhaps just an example). Without that people will have to dig into the code to figure out what change is needed here.

**Comment by pablolmedorado** [comment]
I agree and am willing to change it. The reason why I have not done it from the beginning is because I am not very confident in my level of English. It is my first pull request and I did not know how to proceed.

If you want, I could try to explain it and, if you think there is a better translation, you can correct me.

Best regards.

**Comment by auvipy** [comment]
> I agree and am willing to change it. The reason why I have not done it from the beginning is because I am not very confident in my level of English. It is my first pull request and I did not know how to proceed.
> 
> If you want, I could try to explain it and, if you think there is a better translation, you can correct me.
> 
> Best regards.

sure please

**Comment by rtpg** [comment]
@pablolmedorado that sounds good, yes! It doesn't have to be very long, just a bit more details (given this would be a backwards incompatible change)

Given that it's been a while since the last release, I think once this is cleaned up we can package up a release

**Comment by pablolmedorado** [comment]
I've updated the changelog according to what you pointed out yesterday. Feel free to modify anything you consider wrong.

Thanks for your patience! BR.

**Review by auvipy — APPROVED** [review]

**Review by auvipy — APPROVED** [review]

**Pull request merged**